bindings/wasm/src/account/types/did_type.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use identity_iota::account_storage::DIDType;
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen(js_name = DIDType)]
+#[derive(Clone, Copy, Debug, Deserialize, Serialize)]
+pub enum WasmDIDType {
+  #[serde(rename = "iotaDID")]
+  IotaDID = 1,
```

Comment:
Is the capitalisation correct: `iotaDID` -> `IotaDID`? Is it something to do with Napi serialization?

Is there a reason it uses `1` explicitly; TypeScript enums start at `0`? The Rust definition doesn't specify `1`, so this might cause a mismatch?

```suggestion
  #[serde(rename = "IotaDID")]
  IotaDID,
```